apps/test/unit/templates/sectionProgress/progressTables/ProgressTableSummaryViewTest.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {expect} from '../../../../util/reconfiguredChai';
+import {mount} from 'enzyme';
+import {Provider} from 'react-redux';
+import {createStore, combineReducers} from 'redux';
+import progress from '@cdo/apps/code-studio/progressRedux';
+import sectionData from '@cdo/apps/redux/sectionDataRedux';
+import sectionProgress from '@cdo/apps/templates/sectionProgress/sectionProgressRedux';
+import scriptSelection from '@cdo/apps/redux/scriptSelectionRedux';
+import locales from '@cdo/apps/redux/localesRedux';
+import ProgressTableSummaryView from '@cdo/apps/templates/sectionProgress/progressTables/ProgressTableSummaryView';
+import ProgressTableContainer from '@cdo/apps/templates/sectionProgress/progressTables/ProgressTableContainer';
+import SummaryViewLegend from '@cdo/apps/templates/sectionProgress/progressTables/SummaryViewLegend';
+import ProgressTableSummaryCell from '@cdo/apps/templates/sectionProgress/progressTables/ProgressTableSummaryCell';
+import * as Sticky from 'reactabular-sticky';
+import ProgressTableContentView from '@cdo/apps/templates/sectionProgress/progressTables/ProgressTableContentView';
+import ProgressTableStudentList from '@cdo/apps/templates/sectionProgress/progressTables/ProgressTableStudentList';
+import _ from 'lodash';
+import {
+  fakeLevels,
+  fakeLessonWithLevels,
+  fakeScriptData,
+  fakeStudents,
+  fakeProgressTableReduxInitialState
+} from '@cdo/apps/templates/progress/progressTestHelpers';
+
+const LESSON_1 = fakeLessonWithLevels({position: 1, levels: fakeLevels(1)});
+const LESSON_2 = fakeLessonWithLevels({position: 2, levels: fakeLevels(2)});
+const STAGES = [LESSON_1, LESSON_2];
+const STUDENTS = fakeStudents(2);
+const SCRIPT_DATA = fakeScriptData({stages: STAGES});
+
+const initialState = fakeProgressTableReduxInitialState(
```

Comment:
Passing in data instead of having it generated by `fakeProgressTableReduxInitialState` so that it can more easily be referenced in tests below.